### ISSUE TITLE
SF-Net: Single-Frame Supervision for Temporal Action Localization

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#72]
# TL;DR
They propose SF-Net, which predicts the beginning and end of an action based on a single frame (image). The system not only predicts the class of the action in each frame and the whole video, but also predicts the "action" or "no action" as well. It can be used as an annotation tool.
![Video Frames](https://user-images.githubusercontent.com/24524018/92557663-af232e00-f2a7-11ea-814f-724e1f1bc411.png)
![FERRETS](https://user-images.githubusercontent.com/24524018/92557675-b8ac9600-f2a7-11ea-8810-685f15d99e4e.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2003.06845

# Submission Dates(yyyy/mm/dd)
2020/03/15

# Authors and institutions
Fan Ma, Linchao Zhu, Yi Yang, Shengxin Zha, Gourab Kundu, Matt Feiszli, Zheng Shou
- University of Technology Sydney, Australia
- Facebook

# Methods

# Results

# Comments
ECCV2020
